### ISSUE TITLE
Extra % characters added to Bible verse

### ISSUE THREAD

**Issue by chrisg32** [tim-hub/obsidian-bible-reference#140]
**Describe the bug**
Since the latest round of updates I have percentage characters added to the end of the Bible verse. Is this to be expected?

**To Reproduce**
1. Insert a Bible verse
2. See the extra characters in the markdown

```
> [!Bible] [Hebrews 12:14 - ESV](https://bolls.life/ESV/58/12/)
>  <sup> 14 </sup>Strive for peace with everyone, and for the holiness without which no one will see the Lord.
 %% %%
```

**Expected behavior**
No extra characters are added.

**Screenshots**
![image](https://github.com/tim-hub/obsidian-bible-reference/assets/1832474/28bdaa36-a537-4957-b5f9-202ce0beffbb)

**What Device Do You Use(please complete the following information):**
- Device: Mac
- OS: Ventura 13.5.2
- Obsidian Version 1.4.13 (Installer v1.4.13)
- Version 2.1.2

**Additional context**
Bolls life ESV translation.

**Comment by tim-hub** [comment]
Thanks.

This should be fixed in release 2.1.3 now. 

Please let me know if there is still something wrong.

**Comment by chrisg32** [comment]
@tim-hub as usual thanks for all your hard work.

I am going to make a point to get familiar with the project so that I can contribute to the code/help out where I can but my free time is limited so... hopefully soon 🤞.
